### DISCUSSION TITLE
SEB unable to install

### DISCUSSION THREAD

**Discussion by lawr-cruz** [SafeExamBrowser/seb-win-refactoring#141] [Q&A] [answered]
Good Day. I am trying to install SEB on a new laptop however the install failed. The install appears to stop on an issue regarding Microsoft Edge. I have already installed Webview 2 as well just in case to no change. Here's the log attached below 
[Safe_Exam_Browser_20210803180448.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/6923290/Safe_Exam_Browser_20210803180448.log)

**Comment by dbuechel** [comment, marked as answer]
Microsoft has updated their WebView2 bootstrapper (yet again). The issue will be fixed with the upcoming release of version 3.3.0. For the meantime, first install all the required dependencies as listed below:

- .NET Framework 4.7.2 Runtime: https://dotnet.microsoft.com/download/dotnet-framework/net472
- Microsoft Edge WebView2 Runtime: https://go.microsoft.com/fwlink/p/?LinkId=2124703
- Visual C++ 2015-2019 Redistributable: https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

Then install the SEB 3.2.0 MSI for your platform (either x86 for 32-bit or x64 for 64-bit machines):

- https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/3.2.0/SEB_3.2.0.338_x64_Setup.msi
- https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/3.2.0/SEB_3.2.0.338_x86_Setup.msi

**Reply by lawr-cruz** [reply]
Thank you! That worked.

**Reply by Tindaro16** [reply]
I have already installed a newer NET framework version, the 4.8. Is that a problem? Anyways, when will the 3.3.0 version be released? Because I need SEB to work for a test in september.

**Reply by dbuechel** [reply]
> 
> 
> I have already installed a newer NET framework version, the 4.8. Is that a problem? Anyways, when will the 3.3.0 version be released? Because I need SEB to work for a test in september.

No, that shouldn't be a problem.

**Reply by dbuechel** [reply]
SEB 3.3.0 which fixes this issue is now officially released.

**Reply by ankitada** [reply]
But i am unable to download SEB 3.3.0. Is anyone else facing same issue. It says network error.
![1331_257_1](https://user-images.githubusercontent.com/88391165/128162709-e3f8cef3-310e-4985-845c-15cedc0cdb11.png)

**Reply by dbuechel** [reply]
> 
> 
> But i am unable to download SEB 3.3.0. Is anyone else facing same issue. It says network error.
> ![1331_257_1](https://user-images.githubusercontent.com/88391165/128162709-e3f8cef3-310e-4985-845c-15cedc0cdb11.png)

The download works just fine, so there must be a network issue on your side. And please open a new discussion or issue for questions / problems which don't are directly related with the original topic. We really would like to avoid having the same mess here as we had it over on SourceForge.
